Makefile

```diff
@@ -131,7 +137,7 @@ test-dracpuinfo: ## build helper to expose hardware info in the internal dracpu
 	go build -v -o "$(OUT_DIR)/dracpuinfo" ./test/image/dracpuinfo
 
 test-e2e-base: ## run e2e test base suite
```

Comment:
can we add a log in TestE2E() if KUBECONFIG is passed with "make test-e2e-base" and test is being skipped ?

Reply:
I think I managed to get a better fix and I'm incorporating with the next upload.